app/schemas/models/interactives/common/interactive_types.schema.js

```diff
@@ -0,0 +1,130 @@
+// Specific schemas for different types of interactives
+
+const solutionSchema = require('./solutions.schema')
+const schema = require('../../../schemas')
+
+const interactiveDraggableOrderingSchema = {
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    labels: { type: 'array', items: { type: 'string' } },
+    elements: {
+      type: 'array',
+      items: {
+        type: 'object',
+        additionalProperties: false,
+        properties: {
+          text: { type: 'string' },
+          elementId: schema.objectId({ hidden: true })
+        }
+      }
+    },
+    solution: solutionSchema.elementOrderingSolutionSchema
+  }
+}
+
+const interactiveInsertCodeSchema = {
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    starterCode: { type: 'string' }, // codeLanguage will be determined by unitCodeLanguage in interactives schema
```

Comment:
Should this be an enum of the code options?
There is also a todo in the Voyager Interactives Schema doc that maybe addresses this field?

Reply:
I see it has been addressed below. Ignore comment.